### ISSUE TITLE
Cannot update cache on Windows

### ISSUE THREAD

**Issue by tayopi** [tldr-pages/tldr-node-client#435]
I just installed tldr on my Windows system and no matter what I do, I can't seem to get the cache setup. I've tried using `tldr tar`, `tldr --update`, `tldr --random`, and `tldr --random-example`, but nothing seems to work. I'm not seeing any error messages, so I'm not sure what's going on. Any help would be appreciated.

## Expected behavior
When I type `tldr <command>` or `tldr --update` the cache should be updated.

## Actual behavior
I can't seem to setup the cache. Every time I try nothing seems to happen.

## Log, debug output
![image](https://github.com/tldr-pages/tldr-node-client/assets/2684768/1571e734-3ee0-43bd-9bdc-40d9be9e0abc)

## Environment
- **Operating system** - Windows 10.0.19045
- **Shell** - PowerShell 7.4.1
- **Node.js version** - v21.6.1
- **tldr-node-client version** - v3.4.0
